drivers/SmartThings/zigbee-button/src/zigbee-multi-button/init.lua

```diff
@@ -49,15 +53,23 @@ end
 local function added_handler(self, device)
   local config = supported_values.get_device_parameters(device)
   for _, component in pairs(device.profile.components) do
+    log.debug("### component: " .. utils.stringify_table(component, "component table", true))
```

Comment:
Hey @zwave-mke Can you help me understand why these debug statements are required? Please remove them.